src/main/command_executor.cpp

```diff
@@ -130,42 +132,46 @@ bool CommandExecutor::doCommandSingleton(Command* cmd)
 
   // dump the model/proof/unsat core if option is set
   if (status) {
-    Command* g = NULL;
+    std::vector<std::unique_ptr<Command> > getCommands;
```

Comment:
getCommands -> get_commands

ATM getCommands looks like a function name like "getDumpModels", "getDumpProofs", ...

Reply:
I renamed it to `getterCommands`. Since we use camel-case for variable names, `get_commands` would look a bit out-of-place IMHO.